mofacts/client/views/experiment/card.html

```diff
@@ -1,6 +1,13 @@
 <template name="card">
     <div class="container" id="cardContainer">
-
+    
+    {{#if displayFeedback}}
+        {{>profileDialogueToggles}}
+        <div>
+            <br />
+            <button id='confirmFeedbackSelection' class='btn'>Confirm feedback settings</button>
+        </div>
+    {{/if}}
```

Comment:
I like to make it very explictly clear when two paths diverge.  Can you put the normal card template html into an else condition here so it's very obvious that these are supposed to be separate portions of the card loop?

Reply:
^

Reply:
^